docs/data/data-grid/filtering/QuickFilteringCustomizedGrid.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import {
+  DataGrid,
+  GridCellParams,
+  GridToolbarQuickFilter,
+  GridQuickFilterLogic,
+} from '@mui/x-data-grid';
+
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+function QuickSearchToolbar() {
+  return (
+    <Box
+      sx={{
+        p: 0.5,
+        pb: 0,
+      }}
+    >
+      <GridToolbarQuickFilter
+        quickFilterParser={(searchInput) =>
+          searchInput.split(',').map((value) => value.trim())
+        }
+      />
+    </Box>
+  );
+}
+
+const VISIBLE_FIELDS = ['name', 'rating', 'country', 'dateCreated', 'isAdmin'];
+
+export default function QuickFilteringCustomizedGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Employee',
+    visibleFields: VISIBLE_FIELDS,
+    rowLength: 100,
+  });
+
+  // Otherwise filter will be applied on fields such as the hidden column id
```

Comment:
I don't understand this comment

Reply:
The useDemoData returns by default a column `{field: 'id', hidden: true}`

This column is used in the filtering but not visible. So if you quick filter with value `"45"` it will return some rows because their id contains "45" somewhere in its random characters

I think I will remove this and use the opportunity to show how to deactivate the quick filter on the column `"id"`